v4/export/sql.go

```diff
@@ -315,6 +318,26 @@ func GetColumnTypes(db *sql.Conn, fields, database, table string) ([]*sql.Column
 	return rows.ColumnTypes()
 }
 
+func GetPrimaryKeyColumns(db *sql.Conn, database, table string) ([]string, error) {
+	priKeyColsQuery := "SELECT column_name FROM information_schema.KEY_COLUMN_USAGE " +
+		"WHERE table_schema = ? AND table_name = ? AND CONSTRAINT_NAME = 'PRIMARY' order by ORDINAL_POSITION;"
+	rows, err := db.QueryContext(context.Background(), priKeyColsQuery, database, table)
+	if err != nil {
+		return nil, errors.Annotatef(err, "sql: %s", priKeyColsQuery)
+	}
+	defer rows.Close()
+	var cols []string
+	var col string
+	for rows.Next() {
+		err = rows.Scan(&col)
+		if err != nil {
+			return nil, errors.Annotatef(err, "sql: %s", priKeyColsQuery)
+		}
+		cols = append(cols, col)
+	}
+	return cols, nil
```

Comment:
also check `rows.Err()` before returning.

Reply:
done. Please review it agein.